### ISSUE TITLE
probably a malware

### ISSUE THREAD

**Issue by ilpiruz** [cheat-engine/cheat-engine#2414]
i just installed cheat engine   and i got fps and internet connection quality drop AND I HAVEN'T OPENED THE PROGRAM

**Comment by LouieIsShort** [comment]
**Noob!**

**Comment by ilpiruz** [comment]
> **Noob!**
lmao, leave that malware on your computer and stop being a clown bruh

**Comment by quanmanss11** [comment]
Did you forget, bro, that using "cheatengine" can cause your PC to slow down?

**Comment by cxtal** [comment]
@quanmanss11 you're not wrong. Every program causes your PC to slow down (but just a little!).
